machine_management/more-rhel-compute.adoc

```diff
@@ -6,7 +6,7 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 If your {product-title} cluster already includes Red Hat Enterprise Linux (RHEL)
-compute, or worker, machines, you can add more RHEL compute machines to it.
+worker machines, you can add more RHEL compute machines to it.
```

Comment:
i was under the assumption that we were moving away from `worker` and more towards `compute` ?

Reply:
Thanks for the feedback @abhinavdahiya. Instead of choosing worker over compute, I've instead changed the wording to be clearer (which was the intent of this PR). Let me know if these changes look okay to you - especially the one the one here [1] which is slightly different in that it mentions replicas.

[1] https://github.com/openshift/openshift-docs/pull/20088/files#diff-b81c8ab7e12a1648ebab26d2f023f928R61-R62